aptos-move/framework/aptos-framework/sources/aptos_account.spec.move

```diff
@@ -223,19 +223,4 @@ spec aptos_framework::aptos_account {
         aborts_if type_info::type_of<CoinType>() != type_info::type_of<AptosCoin>()
             && !coin::is_account_registered<CoinType>(to) && !can_receive_direct_coin_transfers(to);
     }
-
-    spec schema TransferEnsures<CoinType> {
```

Comment:
Is this meant to be deleted? It seems to be still used in aptos_account.spec.move:37:17 like the CI lint test complains.

Reply:
I have add it back